feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/captive.c

```diff
@@ -91,65 +98,96 @@ const struct uci_blob_param_list dnsm_param = {
 	.params = dnsm_policy,
 };
 
-void vif_state_dhcp_allowlist_get(struct schema_Wifi_VIF_State *vstate,struct uci_section *s)
+/*void removeSpaces(char *str)
 {
+	int count = 0;
+	for (int i = 0; str[i]; i++)
+		if (str[i] != ' ')
+			str[count++] = str[i];
+	str[count] = '\0';
+}*/
```

Comment:
please just remove unused code rather than comment it out

Reply:
okay